tapdb/sqlc/migrations/000007_universe.up.sql

```diff
@@ -81,3 +81,29 @@ CREATE VIEW universe_stats AS
     FROM universe_events u
     JOIN universe_roots roots ON u.universe_root_id = roots.id
     GROUP BY roots.asset_id, roots.group_key, roots.namespace_root;
+
+-- This table contains general configuration for universe federation syncing.
+CREATE TABLE IF NOT EXISTS federation_general_sync_config(
+    -- This ID column is used to ensure that only one row exists in this table.
+    id INTEGER PRIMARY KEY,
+
+    -- This flag indicates that, in general, only issuance proofs should be
+    -- synced.
+    only_sync_issuance_proofs BOOLEAN NOT NULL
```

Comment:
Also I think we want this value to be true initially? So default is just issuance, then you can flip this (so maybe rename it to accept everything). Then you can create sync configs for sets of assets, and that supercedes this general config?